data.sql

```diff
@@ -31,4 +31,46 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Blossom', '1998-10-13', 17, 'true', 3);
 
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
-VALUES ('Ditto', '2022-05-14', 22, 'true', 4);
\ No newline at end of file
+VALUES ('Ditto', '2022-05-14', 22, 'true', 4);
+
+-- Multiple tables
+INSERT INTO owners (full_name, age) 
+VALUES ('Sam Smith',34 ),
+       ('Jennifer Orwel',19 ),
```

Comment:
- [ ] Please kindly correct the name here, is `Jennifer Orwel` instead of `Jennifer Orwel` 😁 😁